scene/gui/rich_text_label.cpp

```diff
@@ -447,7 +449,9 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 					fh = line_ascent + line_descent;
 
 					if (end && c[end - 1] == ' ') {
-						if (align == ALIGN_FILL) {
+						if (p_mode == PROCESS_CACHE) {
+							spaces_size += font->get_char_size(' ').width;
+						} else if (align == ALIGN_FILL) {
```

Comment:
What's the purpose for unused `spaces_size` variable?

```suggestion
						if (align == ALIGN_FILL && p_mode != PROCESS_CACHE) {
```
This should do  the same, but without unnecessary `get_chat_size` calls.

Reply:
I assumed it would be able to handle `ProcessMode p_mode` without raising errors.